src/main/java/com/google/sps/servlets/FindMentorServlet.java

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+import com.google.sps.data.Mentor;
+import com.hubspot.jinjava.Jinjava;
+import com.hubspot.jinjava.JinjavaConfig;
+import com.hubspot.jinjava.loader.FileLocator;
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(urlPatterns = {"/find-mentor"})
+public class FindMentorServlet extends HttpServlet {
+
+  private static Collection<Mentor> relatedMentors = new ArrayList<>(5);
+
+  @Override
+  public void init() {
+    relatedMentors.add(
+        new Mentor("Alice", "12345", "03/17/2001", "technology", "Master's in Computer Science"));
+    relatedMentors.add(
+        new Mentor("Bob", "23456", "05/23/1985", "medicine", "Master's in Biomedical Engineering"));
+    relatedMentors.add(
+        new Mentor("Charlie", "34567", "07/27/1960", "mental health", "PhD is Psychology"));
```

Comment:
can we move the dummy data to a DummyDataAccess class. This way when you want to clean up dummy data and replace it with data from your db the change is very clean.